sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -231,4 +232,34 @@ export class ApplicationAESTController extends BaseController {
       applicationId,
     );
   }
+
+  /**
+   * Get application overall details for an application.
+   * @param applicationId application Id.
+   * @returns application overall details.
+   */
+  @ApiNotFoundResponse({
+    description: `Application not found or not on ${ApplicationStatus.Completed} status.`,
+  })
```

Comment:
The documentation of not found response doesn't match with the actual. Meaning we expect error which just say `Application not found.`